FileUtility.java

```diff
@@ -0,0 +1,92 @@
+import java.io.*;
+import java.text.SimpleDateFormat;
+
+import javax.swing.*;
+import java.util.*;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.xssf.usermodel.*;
+
+public class FileUtility extends ReDoReport {
+	
+	 static JFileChooser fileChooser;
+	 	static ArrayList<String> projects = new ArrayList<>();
+		static ArrayList<Date> dates = new ArrayList<>();
+		static ArrayList<String> names = new ArrayList<>();
+		static ArrayList<Double> duration = new ArrayList<>();
+	 public static void findFile() {
+		fileChooser = new JFileChooser(".");
```

Comment:
I think you probably want to open the file chooser to the users home directory not the current directory since you can't know where they'll be running this application from.

Reply:
Is the `fileChooser.setCurrentDirectory(new File(System.getProperty("user.home")));` (line 20 below) taking care of the starting directory?

Reply:
fileChooser.setCurrentDirectory(new File(
				System.getProperty("user.home") + System.getProperty("file.separator")));

Yes?

Reply:
I can't tell by inspection. Does it work? Did it work before?